modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -891,9 +891,11 @@
          },
          "THEME": {
            "CAPTION": "Theme",
-           "DESCRIPTION": "",
+           "DESCRIPTION": {
+             "TEXT": "You can optionally use themes to brand your videos. Themes may define bumper and trailer videos, watermarks and title slides including metadata. The brandings defined by the theme will be applied when publishing events of this series."
+           },
            "TITLE": "Theme",
-           "LABEL": "No theme",
+           "LABEL": "Please select theme...",
```

Comment:
Why not … (i.e. U+2026)?

Also many style guides suggest separating ellipses from the surrounding text with a space.

Reply:
+1 for using `…`. We already use an ellipsis in most strings. There was a time when it was used for all strings :-(

> Also many style guides suggest separating ellipses from the surrounding text with a space.

And others don't. Until now, a space character is used nowhere in Opencast and would rather break consistency.

Reply:
Looking at this and other, similar strings, I would probably omit the ellipsis completely and just say `Select theme` or `Select a theme`

Reply:
@lkiesow the other style guides are obviously wrong! :stuck_out_tongue_closed_eyes: 

Seriously though, consistency > style (within reason) and I would actually prefer the version without ellipsis that you braught up, too. :thinking: